AssetTracking/AssetTracking/Models/Sites.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Graph;
+using AssetTracking.Models;
+using AssetTracking.Helpers;
+
+namespace AssetTracking
+{
+    public static class Sites
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="graphServiceClient"></param>
+        /// <param name="groupsId"></param>
+        /// <param name="siteId"></param>
+        /// <param name="listId"></param>
+        /// <returns></returns>
+        public static async Task<Site> GetSites(GraphServiceClient graphServiceClient, string groupsId, string siteId, string listId)
+        {
+            var sites = await graphServiceClient
```

Comment:
Wrap this in a `try catch` block which return meaningful error messages. Do the same for the below resource methods which already don't have this implemented.